Makefile

```diff
@@ -16,7 +16,7 @@ SHELL = /usr/bin/env bash -o pipefail
 .SHELLFLAGS = -ec
 
 .PHONY: all
-all: build
+all: build test lint template docker-build
```

Comment:
Do you need this, since you have semaphore doing the same above? 

Meaning, if I type "make" in my own dev machine, should I be doing all that stuff?